components/encoder/ams/ams_as5048.go

```diff
@@ -124,14 +124,34 @@ func newAS5048Encoder(
 	conf resource.Config,
 	logger logging.Logger,
 ) (encoder.Encoder, error) {
```

Comment:
can we clean up the constructor so it is of the form:
```Constructor: newAmsEncoder```
rather than
```Constructor: func(.....){...}```

Reply:
I agree with the sentiment of your comment, but don't see where it goes. The only mention of `Constructor` in this file already has that form. Did you perhaps mean to put this comment on a different PR?

Reply:
Rand and I discussed: this comment was left by mistake.